### DISCUSSION TITLE
Positive community feedback 💖

### DISCUSSION THREAD

**Discussion by marcoieni** [release-plz/release-plz#1028] [General]
Here, I collect all the nice things people say about release-plz. 
I will look at this when I need a morale boost. 😁

If you want to say "thanks" for release-plz, that's the place!

**Comment by marcoieni** [comment]
> As far as I can tell, release-plz [..] is best-of-class.

--- [fasterthanlime](https://github.com/rust-lang/libc/issues/3350#issuecomment-1746436699)

**Comment by marcoieni** [comment]
> This is a great project to contribute to, it's so incredibly valuable for other maintainers like myself to be able to automate project releases and provide a better experience for our contributors. I'm already using it in [octocrab](https://github.com/XAMPPRocky/octocrab) and plan to adopt it in all my projects eventually and going forward.

--- [XAMPPRocky](https://www.reddit.com/r/rust/comments/13he55f/comment/jkckcx9/?utm_source=share&utm_medium=web2x&context=3)

**Comment by marcoieni** [comment]
> I can vouch for Marco he's a great mentor and release-plz is a great project for new contributors

--- [IngenuityAmazing](https://www.reddit.com/r/rust/comments/13he55f/comment/jkch82i/?utm_source=share&utm_medium=web2x&context=3)

**Comment by marcoieni** [comment]
> Release-plz will automatically run cargo-semver-checks as part of the release process. If you are in the market for a good release manager, you should check this one out; it's awesome.

--- [Predrag](https://fosdem.org/2024/schedule/event/fosdem-2024-2682-semver-in-the-rust-ecosystem-breakage-tooling-and-edge-cases/) (minute 14:00)

**Comment by marvin-hansen** [comment]
Just want to give a big thank you! I realized that my CI runs on release-plz for well over two years and never failed once. In fact, it's so reliable that every single release was a non-event. This project is a bit like git. Yes you can work without it, but you would be very stupid to even try... Keep up the good work and recognize, your work keeps countless projects running.💪

**Reply by marcoieni** [reply]
Thanks Marvin, very kind ❤️ I shared this message on my socials, I hope you don't mind!

**Comment by marvin-hansen** [comment]
Absolutely. 💪
